freezeyt/freezer.py

```diff
@@ -409,6 +409,8 @@ async def prepare(self):
         self.add_task(self.prefix, reason='site root (homepage)')
         for url_part, kind, content_or_path in get_extra_files(self.config):
             if kind == 'content':
+                # join part with path, otherwise filename 'http:' overwrite prefix
```

Comment:
```suggestion
                # join part with path, otherwise filename 'http:' overwrite prefix
                assert self.prefix.path.endswith('/')
                assert not url_part.startswith('/')
```